test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -141,17 +141,27 @@ bootstrap_remote_freebsd()
         "
 
     if [ "${controller}" ]; then
-        jinja2_pkg="py${python_package_version}-jinja2"
-        cryptography_pkg="py${python_package_version}-cryptography"
-        pyyaml_pkg="py${python_package_version}-yaml"
+		jinja2_pkg="py${python_package_version}-jinja2"
+		pyyaml_pkg="py${python_package_version}-yaml"
+		cryptography_pkg="py${python_package_version}-cryptography"
+		packaging_pkg="py${python_package_version}-packaging"
```

Comment:
It looks like some tabs have been mixed in here and a few other places for indentation instead of spaces.